### PR TITLE
fix: Correct the formula for Fahrenheit -> Celsius

### PR DESCRIPTION
**Pull request by richb-hanover**
Correct formula is C = (F-32) / 1.8

### CODE REVIEW

File: book/tests/snapshots/snapshot__run_display_reference_prql@functions-0.prql.snap [hunk 1]
```diff
@@ -3,7 +3,7 @@ source: book/tests/snapshot.rs
 expression: Statements(parse(&prql).unwrap())
 input_file: book/tests/prql/queries/functions-0.prql
 ---
-func celsius_of_fahrenheit temp -> temp - 32 * 3
+func celsius_of_fahrenheit temp -> temp - 32 / 1.8
```
Comment (snth) [inline book/tests/snapshots/snapshot__run_display_reference_prql@functions-0.prql.snap:6]: Does this not need parentheses, i.e. `(temp - 32) / 1.8`?

File: book/tests/snapshots/snapshot__run_display_reference_prql@functions-3.prql.snap [hunk 2]
```diff
@@ -3,7 +3,7 @@ source: book/tests/snapshot.rs
 expression: Statements(parse(&prql).unwrap())
 input_file: book/tests/prql/queries/functions-3.prql
 ---
-func celsius_of_fahrenheit temp -> temp - 32 * 3
+func celsius_of_fahrenheit temp -> temp - 32 / 1.8
```
Comment (snth) [inline book/tests/snapshots/snapshot__run_display_reference_prql@functions-3.prql.snap:6]: Does this not need parentheses, i.e. `(temp - 32) / 1.8`?

File: book/tests/snapshots/snapshot__run_display_reference_prql@functions-4.prql.snap [hunk 3]
```diff
@@ -3,7 +3,7 @@ source: book/tests/snapshot.rs
 expression: Statements(parse(&prql).unwrap())
 input_file: book/tests/prql/queries/functions-4.prql
 ---
-func celsius_of_fahrenheit temp -> temp - 32 * 3
+func celsius_of_fahrenheit temp -> temp - 32 / 1.8
```
Comment (snth) [inline book/tests/snapshots/snapshot__run_display_reference_prql@functions-4.prql.snap:6]: Does this not need parentheses, i.e. `(temp - 32) / 1.8`?

File: book/src/queries/functions.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ Functions have two types of parameters:
 So this function is named `celsius_of_fahrenheit` and has one parameter `temp`:
 
 ```prql
-func celsius_of_fahrenheit temp -> (temp - 32) * 3
+func celsius_of_fahrenheit temp -> (temp - 32) / 1.8
```
Comment (snth) [inline book/src/queries/functions.md:20]: Could you also rename the function to `celsius_from_fahrenheit` or `fahrenheit_to_celsius` while you're at it?

**Review by snth — COMMENTED** [review]
Thanks. I think I corrected this in one or two examples before but not exhaustively. Would be great if we could get these all fixed.

**Comment by richb-hanover** [comment]
Abandoning this - doesn't fix everything

**Pull request closed without merging**